docs/intro.rst

```diff
@@ -0,0 +1,111 @@
+It is capable of incrementally copying data form Kafka into HDFS such that
+every run of the MapReduce job picks up from where the previous run left off.
+At LinkedIn, Camus is used to load billions of messages per day from Kafka into HDFS.
+Confluent's version of Camus integrates with Confluent's schema registry which automatically
+validates the data integrity as it is loaded and ensure compatibility of data stored in HDFS
+as schemas are evolved.
+
+Quickstart
+----------
+
+Assuming you have already installed Confluent Platform. You'll need to specify the location
+of Camus's jar file when running the MapReduce job. If you install Camus via zip/tgz archive,
+you can find Camus's jar under ``share/java/camus/``. If you install Camus via rpm or deb,
+the Camus's jar is under ``/usr/share/java/camus/``.
+
+.. sourcecode:: bash
+
+   # Assuming that a Hadoop cluster is already deployed and you have access to it
+   # Start a testing local Kafka cluster (1 Zookeeper node, 1 Kafka node)
+   $ cd kafka
+   $ bin/zookeeper-server-start.sh config/zookeeper.properties
+   $ bin/kafka-server-start.sh config/server.properties
+   # Create testAvro topic in Kafka
+   $ bin/kafka-topics.sh --create --zookeeper localhost:2181 \
+         --topic testAvro --partitions 1 --replication-factor 1
+   $ cd ..
+
+   # Start Schema Registry. The default schema registry settings
+   # automatically work with the default settings of ZooKeeper and Kafka.
+   $ cd schema-registry
+   $ bin/schema-registry-start config/schema-registry.properties
+   $ cd ..
+
+   # Assuming that you have hadoop on your PATH, and make sure that
+   # schema.registry.url points to the correct address.
+   $ cd camus
+   $ hadoop jar confluent-camus-1.0-shaded.jar com.linkedin.camus.etl.kafka.CamusJob \
+     -D schema.registry.url=http://localhost:8081 -P etc/camus.properties
+
+Installation
+------------
+
+You can download Confluent Platform at .... And follow the instructions to install from
+rpm, deb or archives.
+
+Deployment
+----------
+Camus can be run from the command line. You will need to set some properties either by specifying a
+properties file on the classpath using ``-p`` (filename),or an external properties file using ``-P``
+(path to local file system, or to hdfs),
+or from the command line using ``-D property=value``.
+If the same property is set with multiple methods,
+the order of precedence is command-line properties, external properties file and
+classpath properties file. You can find a list of configurations in section Configuration
+options.
+
+
+.. sourcecode:: bash
+
+   $ hadoop jar confluent-camus-1.0-shaded.jar com.linkedin.camus.etl.kafka.CamusJob
+      -D <property=value>
+      -P <path to external properties file>
+      -p <path to properties file from classpath>
+
+Features
+--------
+
+Camus has the following features: 1. Automatic topic discovery, 2. Avro schema management
```

Comment:
Don't number when you're listing only 3 items directly in a sentence. Also don't capitalize within the sentence unless it's a name (i.e. Avro should be capitalized, automatic and date should not).